src/error_function.js

```diff
@@ -16,17 +16,21 @@ function errorFunction(x) {
     const tau =
         t *
         Math.exp(
-            -Math.pow(x, 2) -
```

Comment:
Mind adding more information to the JSDoc comment above this method to explain which algorithm is used to solve it?

Reply:
I just added. Thanks!